e2e/lib/k8up.bash

```diff
@@ -129,10 +158,64 @@ given_a_running_operator() {
 	echo "✅  A running operator is ready"
 }
 
+given_an_existing_backup() {
+	if [ "${#}" != 2 ]; then
+		errcho "$0 Expected 2 arguments, got ${#}."
+		exit 1
+	fi
+
+	backup_file_name=${1}
+	backup_file_content=${2}
+	given_a_subject "${backup_file_name}" "${backup_file_content}"
+
+	apply definitions/backup
+	wait_until backup/k8up-k8up-backup completed
```

Comment:
same here: the `.reason` field of the condition says something about its success. "completed" condition being true only means it is now finished with some result.

In hindsight, maybe it's best if we stay with the `try` syntax for consistency reasons?